broker_test.go

```diff
@@ -826,6 +835,7 @@ func (s *BrokerSuite) TestPartitionOffset(c *C) {
 						{
 							ID:      1,
 							Offsets: []int64{123, 0},
+							Err:     resErr,
```

Comment:
Yeah that was my concern but I realized that after going through the code more thoroughly. So, I think this makes sense as-written.